### PR TITLE
edit rules implemented (JSON format)

### PR DESCRIPTION
**Pull request by ratmole**
Editing the network rules has been implemented.
nodejs is not my forte but it works!

### CODE REVIEW

**Comment by key-networks** [comment]
@ratmole - thanks a ton! - excellent work!  I just have one comment which I'll put inline in your code.

File: src/controllers/zt.js [hunk 1]
```diff
@@ -284,6 +300,7 @@ exports.member_delete = async function(nwid, id) {
 
 exports.network_easy_setup = async function(nwid,
                                             routes,
+                                            rules,
```
Comment (key-networks) [inline src/controllers/zt.js]: Maybe I'm missing something, but I don't understand what this is here for.

File: src/controllers/zt.js [hunk 2]
```diff
@@ -292,6 +309,7 @@ exports.network_easy_setup = async function(nwid,
     {
       ipAssignmentPools: ipAssignmentPools,
       routes: routes,
+      rules: rules,
```
Comment (key-networks) [inline src/controllers/zt.js]: Likewise with this.

File: src/views/networks.pug [hunk 3]
```diff
@@ -35,6 +37,8 @@ block content
             a(href='/controller/network/' + network.nwid + '/name') #{network.name}
           td
             = network.nwid
+          td
+            a(href='/controller/network/' + network.nwid+'/rules') rules
```
[the remark this answers is not in the document]
Comment (ratmole) [reply src/views/networks.pug]: fine by me

**Comment by ratmole** [comment]
made a mess. will reopen with the proper changes

**Comment by mdPlusPlus** [comment]
Is there any chance this will become finished/merged soon?  
Which work needs still to be done?

**Comment by key-networks** [comment]
Unfortunately the repository that submitted this PR has been deleted.  Not sure why @ratmole did that.

**Pull request closed without merging**